spec/support/pdf_matchers.rb

```diff
@@ -3,30 +3,41 @@
 require "rspec/expectations"
 require "pdf-reader"
 
-RSpec::Matchers.define :match_pdf_content do |fixtures_path_to_expected_pdf|
-  match do |actual|
-    expected = PDF::Reader.new(fixtures_path_to_expected_pdf)
-    @expected = expected.pages.map(&:text).join
-    @expected = @expected.delete("\n").delete(" ").delete("\t")
+module Helpers
```

Comment:
Assuming this will all 💥 when the other PR gets merged in and this is rebased/master is merged in!